packages/projection/src/operators/Mappers/withHandles.ts

```diff
@@ -25,21 +27,34 @@ export interface WithHandles {
 const handleFromAssetId = (assetId: Cardano.AssetId) =>
   Buffer.from(Cardano.AssetId.getAssetName(assetId), 'hex').toString('utf8');
 
+const logger: Logger = dummyLogger;
+
+const isValidHandle = (handle: string) => {
+  const pattern = /^[\w.-]+$/;
```

Comment:
Supported characters are `-._ 0-9 a-z`

Reply:
Yes. That regex covers these cases. This is Eslint optimized version. 😅